backend/usecase/creation.go

```diff
@@ -78,3 +79,88 @@ func (u *CreationUsecase) TextCreation(ctx context.Context, req *domain.TextReq,
 	}
 	return nil
 }
+
+func (u *CreationUsecase) TabComplete(ctx context.Context, req *domain.CompleteReq) (string, error) {
+	// For FIM (Fill in Middle) style completion, we need to handle prefix and suffix
+	if req.Prefix != "" || req.Suffix != "" {
+		// System prompt for article continuation
+		systemPrompt := `
+你是集成在编辑器里的“内联续写”模型。任务是在光标处续写用户的文章，使其自然衔接并提升表达质量。
+
+严格遵循：
```

Comment:
在系统提示中包含了中文注释，建议将系统提示内容提取到配置文件或单独的模板文件中，以提高代码的可维护性和国际化支持。

```suggestion
// 考虑将系统提示提取到配置文件或模板文件中
systemPrompt := loadSystemPromptFromConfig()